tck/agent/README.md

```diff
@@ -0,0 +1,9 @@
+# Sample Agent
```

Comment:
Let's call it `SUT` to be explicit that it follows TCK SUT requirements, let's also link them here in the readme.